db/init.sql

```diff
@@ -2,19 +2,31 @@ GRANT ALL PRIVILEGES ON *.* TO 'user'@'%' IDENTIFIED BY 'secret';
 GRANT ALL PRIVILEGES ON *.* TO 'user'@'localhost' IDENTIFIED BY 'secret';
 FLUSH PRIVILEGES;
 
-create database if not exists `Employees`;
+create database if not exists `NewEmployees`;
 
-USE `Employees`;
+USE `NewEmployees`;
 
 CREATE TABLE IF NOT EXISTS `departments` (
   `department_id` int NOT NULL AUTO_INCREMENT,
+  `departmentdetails_id` int NOT NULL,
```

Comment:
use snake casing to name the column in database.
your name should be department_details_id